src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionCtx.java

```diff
@@ -425,7 +425,16 @@ public void updateQuantization(
         log.debug("Quantization enabled");
         Map<String, Object> quantization = map.get("quantization");
         String family = quantization.get("family").toString();
-        double coefficient = (Double) quantization.get("coefficient");
+        Double coefficient = null;
```

Comment:
I think you can one liner this with:

`Double.valueOf(quantization.getOrDefault("coefficient", 1.0d))`

Otherwise if someone explicitly sets the family to linear this'll still blow up.